integration-testing/election-manager/cypress/e2e/configuration.cy.ts

```diff
@@ -20,7 +20,6 @@ describe('Election Manager and Module Converter MS SEMS configuration', () => {
       { force: true }
     );
     cy.contains('Special Election for Senate 15');
-    cy.contains('0d610ab44c');
```

Comment:
### Election Hash Testing Strategy 6
Don't test it! I don't quite know what the value in testing it is, when the data of the election is tested elsewhere in the UI. Simply removed.